backend/api/projects.py

```diff
@@ -297,20 +293,22 @@ def delete_comments(project_id):
     """
     try:
         manifest = g.projects.find_one({'_id': project_id})
-        if manifest is None:
+        if not manifest:
             raise ApiException("Project not found", 404)
-        del manifest[comments]
-        is_valid = g.validator.is_valid(manifest)
-        if is_valid:
-            g.projects.find_one_and_replace({'_id': project_id}, manifest,
-                                            return_document=ReturnDocument.AFTER)
-            make_response("Success")
-        else:
-            validation_errs = [error for error in
-                               sorted(g.validator.iter_errors(manifest))]
-            if validation_errs is not None:
-                raise ApiException("Validation Error: \n" +
-                                   str(is_valid), 400, validation_errs)
+        if 'comments' in manifest:
+            del manifest['comments']
+            manifest['_id'] = str(project_id)
```

Comment:
we should adjust the validator long term, this is a poor solution but works for now